src/components/AgmExtension/ExtensionRequest.vue

```diff
@@ -69,7 +66,10 @@
         class="px-6 py-7"
         :class="{ 'invalid-section': !extensionRequestValid && showErrors }"
       >
-        <v-row no-gutters>
+        <v-row
+          v-if="!isFirstAgm"
+          no-gutters
+        >
```

Comment:
It would be nice to wrap this row in an expand transition for a smoother UI.

Reply:
Yeah, looks better.

Except the transition isn't smooth on enter...? (Darn Vuetify.) Leo, can you try the scale or slide-y transitions and see if they're smooth on enter and leave?